pkg/exporter/exporter.go

```diff
@@ -154,5 +158,12 @@ func New() *Exporter {
 			Help:      "Number of seconds until certificate expires",
 		},
 			[]string{"path", "issuer", "alg", "version", "subject", "dns_names", "email_addresses", "hostname", "nodename"}),
+		certFailed: prometheus.NewGaugeVec(prometheus.GaugeOpts{
+			Namespace: "ssl_certificate",
+			Subsystem: "expiry",
+			Name:      "failed",
+			Help:      "files that were failed to process",
+		},
+			[]string{"path", "nodename"}),
```

Comment:
I'm sure we can add `hostname` label as well

Reply:
fixed